lightning/src/ln/update_fee_tests.rs

```diff
@@ -526,6 +541,14 @@ pub fn test_update_fee_that_funder_cannot_afford() {
 	check_closed_event!(nodes[1], 1, reason, [node_a_id], channel_value);
 }
 
+#[xtest(feature = "_externalize_tests")]
```

Comment:
Are the `update_fee_test` and `channel.rs` changes intentional? It seems like they could use their own commit if so

Reply:
I have really no idea how that slipped through. Reverted.